kolibri/core/assets/src/views/ExamReport/TriesOverview.vue

```diff
@@ -115,30 +115,25 @@
           return 0.0;
         }
       },
+      // Returns the time spent on the best attempt or null if there are no attempts
       bestTimeSpent() {
-        const bestScoreAttempt = this.pastTries.find(t => t.correct === this.maxQuestionsCorrect);
-        if (!bestScoreAttempt) {
-          return null;
-        }
-        return bestScoreAttempt.time_spent;
+        return this.pastTries.length
+          ? this.pastTries.find(t => t.correct === this.maxQuestionsCorrect).time_spent
+          : null;
```

Comment:
If the length of `pastTries` is not `0`, then we are guaranteed to have a `try` object as the result of `line 121` as `this.maxQuestionsCorrect` would be returning the score of the same (and thus we would always find it). 

If not, then we can return the same as `null`.

Reply:
@EshaanAgg sorry, I just noticed that If, for some reason, there is a corrupted object, and a `try.correct` is `undefined`, then `Math.max` will give `NaN`, and the `.find` will be null. It is better to add the null check, as before, in case anything can happen with the tries objects.